drivers/npmx/npmx_shell.c

```diff
@@ -570,6 +570,83 @@ static int cmd_charger_trickle_set(const struct shell *shell, size_t argc, char
 	return 0;
 }
 
+static int cmd_charger_discharging_current_get(const struct shell *shell, size_t argc, char **argv)
```

Comment:
Either those new command functions should be after functions for `ntc_resistance` or `discharging_current` should be moved above `die_temp` in creating level 2 subcommands for the charger. Now the ordering does not match.